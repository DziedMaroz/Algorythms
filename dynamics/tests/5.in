62806
755
544
55701
530
5808
80462
97551
6811
62675
4
25778
5660
73
9999
5
27
8807
79
92983
167
5
9972
9858
702
6942
2
331
1078
0158
20
48719
24
93013
576
738
70
9061
6
66
88
60
29891
14059
9
768
7876
754
845
543
520
2
2
41836
74033
472
74572
61957
40376
6
64127
43
7
03992
7
4061
26180
3
50
7
340
2857
774
32788
63762
4
85
21579
36
7
8
37222
666
40
3202
1
1016
65
5
5687
7
74469
4119
4655
92690
185
90546
210
5
62
73739
9
54365
63550
653
409
231
85673
64081
520
730
47
8969
3
06511
00299
8482
54991
647
67
770
326
94260
08
145
82831
7504
156
724
96
590
1
4
05
049
2
6928
003
7
6
598
2
4808
59
37400
75198
78
9
4
446
2
65169
7
15
08346
0
30
724
92208
994
3
28
4896
5066
35
933
31
15233
1684
14002
86310
6750
160
716
30829
494
2
92098
3387
8305
8
70238
63
406
45077
453
244
020
08120
87940
1
40544
0478
70133
40487
1
1345
1095
6
369
1650
14
41
955
58531
99699
57
74
020
4
96
062
3
08859
71653
0401
28
6630
907
89
3374
637
8
7
3116
2
02243
81948
405
2557
38375
47
09468
02864
176
19809
0328
767
878
8946
16528
36
2509
8682
41
13
013
389
6651
9972
101
562
039
187
872
035
6
417
2
79
9
9197
10607
73218
3608
34
06
4990
03
70313
2
58433
80295
548
9661
5
85250
74
33
11305
594
49182
57797
79
693
65
627
60340
4
8076
5290
03
487
062
8
6877
73
7
24
7333
94
007
69235
95195
5855
3
13732
347
82586
21255
32
862
8181
76
81
4709
309
5
55953
755
7
439
7046
69
07749
17182
6
34821
2
371
9298
1936
6
470
9386
503
29
0
87
4
9676
66717
118
52
93
845
92843
83662
10
17822
5478
3790
15695
07080
8670
64
8019
6
7
5
23
82
6009
1
30108
75
7898
3504
4
9
4636
01720
6591
504
82284
20329
5
12757
7505
1
5
62558
78
865
16
85143
78
8
9
8
4
01
06
2481
8
7
2
3584
744
7
421
88781
1
2
11394
5771
01525
53258
54
35930
7137
50
311
40271
047
4188
679
502
63031
236
70502
6
381
338
34739
19
8
33009
80
158
4
908
31
50
5
44
08
512
9613
707
552
7
6035
9
094
2536
6
540
35871
50
7018
60
938
08817
20
4434
9
5117
9
613
0853
66960
5487
0998
328
911
0
93
7
4
515
94784
5
12166
14871
24
128
18507
86587
43
5869
28
807
21
79
956
59
2264
8
5122
060
13403
722
52
35170
0283
14188
26566
3
95669
8365
64
652
787
020
0
197
8820
82089
29
805
6
9
6984
1
9289
22
54300
66015
432
3
73873
39
433
1429
81423
70
53
9644
58961
43
518
3
90
61977
10
1
153
4
5
2897
87242
44680
314
48558
9287
2
29
958
1
41033
88113
5929
5273
1
1
2
903
45187
6040
7698
69907
6
8919
1191
6045
2
08243
932
5
97098
6163
72836
1387
05392
634
34634
453
5
85626
25064
131
68
301
6
1
3
4625
2956
62067
22
567
4
38300
21
1
937
005
588
8929
8
87
2
48
72721
111
4
2371
33
60944
67886
411
36
718
5453
47921
4964
96195
7594
51614
6187
1438
21
2
994
8
7
75636
3372
662
97634
0
79
3
4042
94
32
395
1
6894
9
7333
37169
72
3
5
7355
90593
0
19
84465
9
0440
92520
181
48
30
437
96152
8
13
940
6
03738
0278
1271
868
5399
7198
18
166
8437
2551
86790
97278
8910
082
822
4
7244
511
1765
0
02
63197
46055
91418
5020
4
63
48126
9
48343
2127
72
582
994
40596
21150
785
32
31242
7
6357
354
0786
08709
72350
54903
554
28199
8
33
6785
4
353
9
89
72
7
8407
3417
86
519
82
4736
4
113
91
030
272
24
49219
5979
0223
20464
4
1796
41087
6562
24
5015
4
3
1338
881
8
46
9030
006
4247
41850
39595
5
79
7
1711
177
