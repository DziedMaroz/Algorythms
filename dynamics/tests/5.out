3
6 28 06
